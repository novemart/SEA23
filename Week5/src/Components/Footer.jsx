import React from 'react'

const Footer = () => {
  return (
    <footer className='mt-auto py-3 text-center container'>&copy; QA Ltd 2019-</footer>
  )
}

export default Footer