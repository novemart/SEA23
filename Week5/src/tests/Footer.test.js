import React from "react";
import Footer from "../Components/Footer";
import { create } from "react-test-renderer";


test('Footer matches snapshot', ()=>{
    const footer = create(<Footer />)
    expect (footer.toJSON()).toMatchSnapshot();
})