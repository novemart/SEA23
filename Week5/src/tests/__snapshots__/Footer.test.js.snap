// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
<footer
  className="mt-auto py-3 text-center container"
>
  Â© QA Ltd 2019-
</footer>
`;

exports[`Header matches snapshot 1`] = `
<header>
  <nav
    className="navbar navbar-expand-sm"
  >
    <a
      className="navbar-brand"
      href="https://www.qa.com"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt="QA Ltd"
        src="qalogo.svg"
        width="100"
      />
    </a>
    <a
      className="navbar-brand"
      href="/"
    >
      Todo App
    </a>
  </nav>
</header>
`;
